backend/entityservice/views/run/results.py

```diff
@@ -46,9 +48,15 @@ def get_result(dbinstance, project_id, run_id, token):
         return {"groups": result}
 
     elif result_type == 'similarity_scores':
-        logger.info("Similarity result being returned")
-        return get_similarity_score_result(dbinstance, run_id)
-
+        if 'INTERNAL_OBJECT_STORE_ADDRESS' in request.headers:
```

Comment:
I wonder if we can find a more descriptive header. Maybe `RETURN_OBJECT_STORE_ADDRESS`?

Reply:
Sure that would be clearer, or would you think a a separate rest endpoint would be better?